src/metrics/metrics.service.lexi.js

```diff
@@ -1,23 +1,57 @@
 angular
     .module("ovh-api-services")
     .service("MetricsLexi", function ($resource, $cacheFactory) {
+        "use strict";
 
+        var cache = $cacheFactory("MetricsLexi");
         var queryCache = $cacheFactory("MetricsLexiQuery");
-        var r = $resource("/metrics", {}, {
+        var interceptor = {
+            response: function (response) {
+                cache.removeAll();
+                queryCache.removeAll();
+                return response.data;
+            }
+        };
+        var resource = $resource("/metrics/:serviceName", {
+            serviceName: "@serviceName"
+        }, {
             query: {
                 method: "GET",
                 cache: queryCache,
                 isArray: true
+            },
+            get: {
+                method: "GET",
```

Comment:
for readability's sake I think simple API method could be declared as inline objects